hw/ip/otbn/code-snippets/otbn_bn_mul64.asm

```diff
@@ -0,0 +1,549 @@
+/*
+    Copyright lowRISC contributors.
+    Licensed under the Apache License, Version 2.0, see LICENSE for details.
+    SPDX-License-Identifier: Apache-2.0
+
+    This code is a modified version of code released with Chromium OS,
+	which came with the following copyright notice:
```

Comment:
Tab -> Space

Reply:
Thanks. M-x untabify to the rescue... :-)